cypress-setup.sh

```diff
@@ -14,19 +14,27 @@ else
   _PRISMIC_REPO=$REPOSITORY
 fi
 
-if [[ "$PRISMIC_URL" == 'https://wroom-qa.com' ]] || [[ "$PRISMIC_URL" == 'http://wroom.test' ]] ; then 
-  MODE="dev"
+if [[ "$PRISMIC_URL" == 'https://wroom-qa.com' ]] || [[ "$PRISMIC_URL" == 'http://wroom.test' ]] ; then
+  export SM_ENV="development"
 elif [[ "$PRISMIC_URL" == 'https://wroom.io' ]]; then
-  MODE="stage"
+  export SM_ENV="staging"
 else
-  MODE="prod"
+  export SM_ENV="production"
 fi
 
+# TODO: Let adapters decalre their dependencies and automatically install
```

Comment:
Interesting

Reply:
We can remove that temporary line now that `@slicemachine/adapter-next` can install its own dependencies.

The script was updated in this commit: https://github.com/prismicio/slice-machine/commit/87167c9b63b46e3446625901373263336c6a816c

You can see where `@slicemachine/adapter-next` installs dependencies here: https://github.com/prismicio/slicemachine-adapter-next/blob/ca847e227b050d41dd9b4a3b4a465aab5e724319/src/hooks/command-init.ts#L24-L38